apps/src/templates/PuzzleRatingButtons.jsx

```diff
@@ -57,6 +69,10 @@ class PuzzleRatingButtons extends Component {
     };
   }
 
+  static propTypes = {
+    useLegacyStyles: React.PropTypes.bool,
```

Comment:
@markabarrett The assets I got from [here](https://drive.google.com/drive/folders/0B7-SvWwqBhOxS0hRWEM3V3k2TFU) were black, not purple. I checked with Ryan and tried to make purple, but somehow messed up the arrow version. Do you have purple and white versions I can use?

I wasn't planning to change the review buttons, I was just showing how the buttons from the gamification dialog would look here.